app/code/core/Mage/Adminhtml/Block/Sales/Order/View.php

```diff
@@ -375,4 +375,25 @@ public function getHeaderHtml()
     {
         return '<h3 class="' . $this->getHeaderCssClass() . '">' . $this->escapeHtml($this->getHeaderText()) . '</h3>';
     }
+
+
+    public function canDisplayGiftmessageItems()
+    {
+        if (!Mage::helper('core')->isModuleEnabled('Mage_GiftMessage')) {
+            return false;
+        }
```

Comment:
Why not check for module enabled in the function `Mage_GiftMessage_Helper_Message::getIsMessagesAvailable()`?

Reply:
Because if Mage_Giftmesage module is disabled (or doesn't exist),
then searching for Mage_GiftMessage_Helper_Message causes:

```
Error: Class "Mage_Giftmessage_Helper_Message" not found in /var/www/app/Mage.php:612
Stack trace:
#0 /var/www/app/code/core/Mage/Core/Model/Layout.php(625): Mage::helper()
```

Reply:
We can check `Mage::helper('giftmessage/message') !== false`.

Reply:
> We can check `Mage::helper('giftmessage/message') !== false`.

No, as it looks for giftmessage-helper ....

> Fatal error: Uncaught Error: Class 'Mage_Giftmessage_Helper_Message' not found